packages/core/src/types/redocly-yaml.ts

```diff
@@ -313,6 +314,8 @@ const createConfigApisProperties = (nodeTypes: Record<string, NodeType>): NodeTy
   ...nodeTypes['rootRedoclyConfigSchema.apis_additionalProperties'],
   properties: {
     ...nodeTypes['rootRedoclyConfigSchema.apis_additionalProperties']?.properties,
+    root: { type: 'string' },
+    output: { type: 'string' },
```

Comment:
Those two should have come from the [config](https://github.com/Redocly/redocly/blob/main/packages/redocly-config/src/root-config-schema.ts#L138) itself. If they are missing, then something is wrong on Realm's side.

Reply:
You are right. it was a bug in config.